package.json

```diff
@@ -150,6 +151,7 @@
     "nyc": "^13.0.0",
     "postcss": "^7.0.0",
     "prettier": "^1.8.2",
+    "prop-types": "^15.6.0",
```

Comment:
`prop-types` shouldn't be a peer dependency as per recommendation anyway: https://github.com/facebook/prop-types#how-to-depend-on-this-package

This is a workaround for libraries not following this recommendation (e.g. material-ui-pickers)
/cc @dmtrKovalenko

Reply:
Well spotted! We should fix that.